clients/gtest/trmm_gtest.yaml

```diff
@@ -92,15 +97,25 @@ Definitions:
     - { M:  44928,  N: 384,  lda: 44928, ldb:   44928 }
     - { M:  53376,  N: 384,  lda: 53376, ldb:   53376 }
 
-
 Tests:
+- name: trmm_small
+  category: quick
+  function: trmm
```

Comment:
if we combine the alpha ranges then we don't need separate categories here

Reply:
I think keeping them separate will reduce the number of tests. We have 2 values in alpha_range, and 3 values in complex_alpha renge.